agents/deep_research_agent.py

```
@@ -3,279 +3,305 @@
+research to create comprehensive reports at the level of a research analyst.
 
 Features:
-- Uses o4-mini-deep-research-2025-06-26 for cost-effective deep research
+- Uses o4-mini for cost-effective deep research
 - GPT-4.1 priming to enhance research quality
 - Background mode for long-running analysis
 - Isolated functionality for future model expansion
+- LiteLLM Router with automatic fallbacks
 """
 
 import json
```

Ruff (0.12.2)

`json` imported but unused

Remove unused import: `json`

(F401)